numba/tests/test_sort.py

```diff
@@ -715,6 +716,79 @@ class TestQuicksortArrays(BaseQuicksortTest, TestCase):
     def array_factory(self, lst):
         return np.array(lst, dtype=np.float64)
 
+class TestQuicksortMultidimensionalArrays(BaseSortingTest, TestCase):
+
+    quicksort = make_jit_quicksort(is_np_array=True)
+    make_quicksort = staticmethod(make_jit_quicksort)
+
+    def assertSorted(self, orig, result, last_index=None):
+        # sorted() returns a list, so make sure we compare to another list
+        if last_index is not None:
+            self.assertTrue((orig == result[0][:last_index]).all())
+        else:
+            self.assertEqual(orig.shape, result.shape)
+            self.assertTrue((orig == result).all())
+
+    def array_factory(self, lst):
+        array = np.array(lst, dtype=np.float64)
+        return array.reshape(-1, array.shape[0])
```

Comment:
Are we only testing on arrays with shape `(1, n)`? There needs to be test for non-one dimension and on various number of dimension.

Reply:
Updated. Now for a give size, I test on all possible shapes for that size.